tests/cases/fourslash/codeFixPromiseToAsyncCatchAndRej.ts

```diff
@@ -0,0 +1,34 @@
+/// <reference path='fourslash.ts' />
+
+// @target: es6
+
+////function [|f|]():Promise<void> {
+////    return fetch('http://yahoo.com').then(result => { console.log(result); }, rejection => { console.log("rejected:", rejection); }).catch(err => { console.log(err) });
+////}
+
+verify.getSuggestionDiagnostics([{
+    message: "This may be converted to use async and await.",
+    code: 80006,
+}]);
+
+
+verify.codeFix({
+    description: "Convert to use async and await",
+    index: 0,
+    newFileContent:
+`async function f():Promise<void> {
+    try {
+        let result;
+        try {
+            result = await fetch('http://yahoo.com');
+        }
+        catch (rejection) {

```

Comment:
Would people prefer to see `e` here, as is the predominant convention for catch blocks, or the identifier they used in the promise chain? I might lean toward the former, but maybe the latter (what you have) makes sense...